PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -174,26 +176,68 @@ export default {
       minDateStr: "1900-01-01",
       birthdateRules: [
         (v) => !!v || "DOB is required",
-        (v) => v.length == 10 || "DOB must be in specified format",
+        // check if v exists before seeing if the length is 10
+        (v) => !!v && v.length === 10 || "DOB must be in specified format, MM/DD/YYYY",
         (v) => this.validBirthdate(v) || "Invalid DOB",
       ],
     };
   },
+  watch: {
+    date () {
+      this.dateFormatted = this.formatDate(this.date)
+    },
+  },
+  computed: {
+    maxDateStr: function() {
+      let d = new Date();
+      let date = [
+        d.getFullYear(),
+        ("0" + (d.getMonth() + 1)).slice(-2),
+        ("0" + d.getDate()).slice(-2),
+        ].join("-");
+
+      return date;
+    },
+  },
   methods: {
     validBirthdate(birthdate) {
       var minDate = Date.parse(this.minDateStr);
       var maxDate = Date.parse(this.maxDateStr);
-      var date = Date.parse(birthdate);
+      var formattedDate = () => {
+        // Ensure birthdate is fully entered and can be converted into 3 variables
+        if(birthdate) {
+          if(birthdate.split('/').length === 3) {
+            const [month, day, year] = birthdate.split('/');
+            return `${year}-${month.padStart(2, '0')}-${day.padStart(2, '0')}`;
+          }
+        }
+        return false;
+      }
+      var date = Date.parse(formattedDate());
```

Comment:
I would prefer we keep variable names to be representative of their data and specific for what they represent, so maybe `dateOfBirth` here or something?

Reply:
I'm really not sure what the source of the date picker not coming up is. 
I had that happen once while I was developing, then never again after a refresh.

I'll try to see if I can reproduce.

Reply:
I mainly called it date as naming the variable DOB was causing issues for me that I couldn't figure out quickly. I can dig around some more later.